Source/Events/AggregateRoot.cs

```diff
@@ -183,7 +183,7 @@ void ThrowIfAggreggateRootVersionIsOutOfOrder(CommittedAggregateEvent @event)
             if (@event.AggregateRootVersion != Version) throw new AggregateRootVersionIsOutOfOrder(@event.AggregateRootVersion, Version);
         }
 
-        void ThrowIfStateful()
+        void ThrowIfStatefull()
```

Comment:
It was correct before :) 

```suggestion
        void ThrowIfStateful()
```